src/python/grpcio/grpc/__init__.py

```diff
@@ -1788,6 +1843,12 @@ def _create_servicer_context(rpc_event, state, request_deserializer):
     context._finalize_state()  # pylint: disable=protected-access
 
 
+class Compression(object):
```

Comment:
Should we make it an `enum.Enum`? Also, should we add a docstring to this class?

Reply:
Also, can you add this class to Sphinx template located at `doc/python/sphinx/grpc.rst`?